libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -879,6 +879,18 @@ void AP_Logger::periodic_tasks() {
     handle_log_send();
 #endif
     FOR_EACH_BACKEND(periodic_tasks());
+
+    // check for dropped packets once per 10 seconds
+    uint32_t now = AP_HAL::millis();
+    if (now - _last_dropped_check > 10000) {
+        for (uint8_t i=0; i<_next_backend; i++) {
+            uint32_t dropped = backends[i]->dropped_prev_second_total();
+            if (dropped > 50) {
+                GCS_SEND_TEXT(MAV_SEVERITY_WARNING, "Dataflash log dropping %u pkts/sec. Reduce LOG_BITMASK", (unsigned)dropped);
```

Comment:
```suggestion
                GCS_SEND_TEXT(MAV_SEVERITY_WARNING, "Logging: dropping %u pkts/sec. Reduce LOG_BITMASK", (unsigned)dropped);
```

we're not supposed to use "dataflash" any more.  It's rather incorrect now-adays...